apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -481,6 +471,7 @@ where
                     );
                     stats.increment_errored_txs();
 
+                    self.wl_storage.drop_tx();
```

Comment:
Does this only drop the current tx or does it clear the entire wl_storage? From reading the wl_storage code I'm not entirely sure.

Reply:
only the current tx